OpenMEEG/libs/OpenMEEG/src/assembleSensors.cpp

```diff
@@ -117,6 +117,7 @@ namespace OpenMEEG {
         unsigned p0_p1_size = (geo.size() - geo.nb_current_barrier_triangles());
 
         Matrix FergusonMat(3*nbIntegrationPoints, geo.nb_vertices());
+        FergusonMat.set(0.0);
```

Comment:
Strange but the correction seems OK. Probably before, it was using a portion of memory that was never used by the system (and thus set to zero). Somehow, now this reuses some memory already used hence the poor initialisation.